src/components/Results.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from "react";
+import TableHead from './TableHead';
+import TableBody from './TableBody';
+import * as FakeBookings from "../data/fakeBookings.json";
+
+
+class Results extends React.Component {
+	constructor(props) {
+		super(props)
+
+		this.state = {
```

Comment:
Using state in `Results` works for now - because we're just listening for a click in the results table. But in the future when we start trying to do a real search with the input in the `Search` component, I think you'll find that pulling the state up to the `Bookings` component is better.

Reply:
that's not so obvious for me now

Reply:
Ok, try changing your code so that when you click on one of the search buttons, it loads your results. This is more like what we actually would want to make in a real application - show no results until the user searches for something.

You'll probably find that it's quite difficult if you keep the state in `Results`.

Reply:
yeah, i got it, i have state in results because the button i click (i created it) belongs to results block. once we use input fields, which belong to another sibling component, i will want to set state to a parent block